omnibus/config/projects/agentless-scanner.rb

```diff
@@ -0,0 +1,184 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https:#www.datadoghq.com/).
+# Copyright 2016-present Datadog, Inc.
+
+require "./lib/ostools.rb"
+
+name 'agentless-scanner'
+package_name 'datadog-agentless-scanner'
+homepage 'http://www.datadoghq.com'
+license "Apache-2.0"
+license_file "../LICENSE"
+
+if ohai['platform'] == "windows"
+  # Note: this is the path used by Omnibus to build the agent, the final install
+  # dir will be determined by the Windows installer. This path must not contain
+  # spaces because Omnibus doesn't quote the Git commands it launches.
+  install_dir "C:/opt/datadog/agentless-scanner"
+  maintainer 'Datadog Inc.' # Windows doesn't want our e-mail address :(
+else
+  install_dir '/opt/datadog/agentless-scanner'
+  if redhat_target? || suse_target?
+    maintainer 'Datadog, Inc <package@datadoghq.com>'
+
+    # NOTE: with script dependencies, we only care about preinst/postinst/posttrans,
+    # because these would be used in a kickstart during package installation phase.
+    # All of the packages that we depend on in prerm/postrm scripts always have to be
+    # installed on all distros that we support, so we don't have to depend on them
+    # explicitly.
+
+    # postinst and posttrans scripts use a subset of preinst script deps, so we don't
+    # have to list them, because they'll already be there because of preinst
+    runtime_script_dependency :pre, "coreutils"
+    runtime_script_dependency :pre, "grep"
+    if redhat_target?
```

Comment:
IIRC you mentionned that the agentless scanner depends on the agent but this doesn't appear to be reflected here. It would also probably simplify the package scripts quite a bit since most folders & user setup could be handled by the agent's package script